src/details/ArborX_DetailsUtils.hpp

```diff
@@ -581,7 +604,8 @@ void computeOffsetsInOrderedView(ExecutionSpace const &exec_space, View view,
         }
       },
       num_offsets);
-  Kokkos::resize(offsets, num_offsets);
+  Kokkos::resize(Kokkos::view_alloc(exec_space, Kokkos::WithoutInitializing),
+                 offsets, num_offsets);
```

Comment:
This seems like a bug. We want to resize the `offsets` array, but keep its values.

Reply:
It avoids initializing the new elements.